tests/unit/conftest.py

```diff
@@ -0,0 +1,44 @@
+import os
```

Comment:
Yes, [link](https://docs.pytest.org/en/6.2.x/fixture.html#conftest-py-sharing-fixtures-across-multiple-files)

I've created a merged conftest.py that contains fixtures shared by both unit and integration, it sits in the test folder.  However, because the integration and unit test uses different mock directories they still both have their own conftest with fixtures pointing to their respective mock tests.